daffodil-schematron/src/main/resources/iso-schematron-xslt2/schematron-skeleton-api.htm

```diff
@@ -0,0 +1,757 @@
+<!--

```

Comment:
Do we need this file in our schematron embodiment. Can we instead just reference doc in some location for iso schematron?

Reply:
Looks like this content is available at http://schematron.com/schematron-skeleton-api/

I dont see anything that would require those two files to remain in the resources, will add them in the project level readme and remove the files.